doc/apple2.sgml

```diff
@@ -452,10 +452,15 @@ The names in the parentheses denote the symbols to be used for static linking of
   (RTS/CTS) and does interrupt driven receives. Speeds faster than 9600 baud
   aren't reachable because the ROM and ProDOS IRQ handlers are too slow.
   Software flow control (XON/XOFF) is not supported.
+
   Note that because of the peculiarities of the 6551 chip transmits are not
   interrupt driven, and the transceiver blocks if the receiver asserts
   flow control because of a full buffer.
 
+  Note that using the driver at SER_BAUD_115200 will disable IRQs. It will be up
+  to the user to use the serial port, either by re-enabling IRQs themselves,
```

Comment:
I'm no native speaker but shouldn't it either be "users" ... "themselves" or "user" ... "himself" ?

Reply:
I'm no native speaker either, but I was aiming for the non-gendered singular they (https://en.wikipedia.org/wiki/Singular_they).
I switched to user**s** so that other non-native speakers don't think we can't english properly ;)

Reply:
Not a native English speaker here either, but I think the proper way would be "user ... themself"

Reply:
Thanks. At least I would have understood this is to be related to that thing.